pkg/schedule/schedulers/OWNERS

```diff
@@ -1,7 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
remove it?

Reply:
6.5 doesn't have pkg/schedule/schedulers